chia/full_node/full_node.py

```diff
@@ -121,6 +122,17 @@ async def _start(self):
         self.new_peak_sem = asyncio.Semaphore(8)
         # create the store (db) and full node instance
         self.connection = await aiosqlite.connect(self.db_path)
+        if self.config.get("log_sqlite_cmds", False):
+            sql_log_path = path_from_root(self.root_path, "log/sql.log")
+            self.log.info(f"logging SQL commands to {sql_log_path}")
+
+            def sql_trace_callback(req: str):
+                timestamp = datetime.now().strftime("%H:%M:%S.%f")
+                log = open(sql_log_path, "a")
```

Comment:
this callback is called in a separate thread, so we can't capture a file descriptor and pass it in. That's why the *path* is captured by the callback and the file is opened every time. Fortunately that's very fast (on linux)